config/application.rb

```diff
@@ -50,6 +50,10 @@ def allow_invitation_requests?
     true
   end
 
+  def allow_invitation_sendings?
+    true
```

Comment:
non riusciamo a far caricare questa cosa da un file di config o da qualche parte nel DB? solo se viene facile eh, perché altrimenti dobbiamo ricreare il container ogni volta.

Reply:
Metterlo in una variabile di sistema è troppo confusionario?

Reply:
Si può mettere in un file di configurazione yml oppure in una variabile d'ambiente (si potrebbe ovviamente mettere nel database ma mi sembra una soluzione poco pulita). Se creo un file di configurazione poi è possibile modificarlo senza dover ricreare il container?

Reply:
Penso dipenda dalla modalità di esecuzione del server. In development mode sì, altrimenti no. Tipo le traduzioni le ricarica ad ogni esecuzione ma mi aspetto che in production mode non lo faccia. Però questo credo sia un dettaglio di Rails. Prova a vedere qual è il pattern consigliato

Reply:
Allora, da quello che ho visto sia file di configurazione che variabili d'ambiente richiedono che il server sia riavviato. Dato che nel database ogni utente ha il proprio flag `can_invite`, che ne direste invece di due piccoli script (ad es: `enable_invites` e `disable_invites`) che vanno ad abilitare/disabilitare gli inviti per ogni utente? Non credo sia una soluzione molto performante, però ci consente di non dover riavviare il server e ricreare il container, e comunque non sono script che verranno eseguiti di frequente.

Reply:
il problema è che se nel frattempo disabilitiamo gli inviti a qualcuno perché li sta spammando, così glieli ripristiniamo. Inoltre noi admin dovremmo sempre poter invitare.  @ndrkltsk @karjudev @CapacitorSet  idee?

Reply:
Allora io opterei o per lasciare così com'è, oppure per spostare i due flag in un file di configurazione yml. Ricordo che in entrambi i casi il server è da riavviare.

Le variabili d'ambiente invece [non sono consigliate in production](https://support.cloud.engineyard.com/hc/en-us/articles/205407508-Environment-Variables-and-Why-You-Shouldn-t-Use-Them).

Reply:
perché no le variabili d'ambiente? non sono consigliate da chi?

Reply:
Nell'articolo che ho linkato sopra si fa proprio l'esempio di Unicorn, dicendo che il processo master dev'essere completamente terminato affinché le nuove variabili d'ambiente diventino disponibili. In questo modo è più lento riavviare il server.

Inoltre avevamo deciso di seguire la convenzione adottata da lobsters. Se però qualcuno ha idee migliori ben venga.

Reply:
ah ok, no per conformità a lobster va bene il file. Basta che non siano hardcoded e va tutto bene